frontend/src/views/cancellations.js

```diff
@@ -37,7 +42,7 @@ function FlightDetails() {
     seatsStatus > 0 ? (seatsStatus * underageCost) / 1000 : 0
   );
   const [netRevenue, setNetRevenue] = React.useState(
-    currentTotalRevenue - currentOverbookingCost - currentUnderageCost
+    currentTotalRevenue - currentOverbookingCost
```

Comment:
I'm not sure this should be a state, the `netRevenue` is a computed data from another state, instead of saving it in state just define the constant and computed every render.

Reply:
We were just discussing that. We were going to change it anyways.